src/AppEntry.js

```diff
@@ -7,7 +7,9 @@ import { getBaseName } from '@redhat-cloud-services/frontend-components-utilitie
 
 const AppEntry = ({ logger }) => (
   <Provider store={(logger ? init(logger) : init()).getStore()}>
-    <Router basename={`${getBaseName(window.location.pathname, 3)}`}>
+    <Router
+      basename={`${getBaseName(window.location.pathname, 2 + location.href.includes('insights'))}`}
```

Comment:
@karelhala instead of coercing this boolean to an int, could this be updated to do something like the following for clarity?
```suggestion
      basename={`${getBaseName(window.location.pathname, location.href.includes('insights') ? 3 : 2)}`}
```

Reply:
Sure thing!